listeners.py

```diff
@@ -50,7 +57,48 @@ def process_result(self, obj):
                                       obj.result_checksum,
                                       obj.show_reproduce_content))
 
+    def get_failed_task_name(self, task):
+        return task[0][0] + ((':' + task[0][1]) if task[0][1] else '')
+
+    def get_long_mark(self, task):
+        return '(long)' if task in self.long_tasks else ''
+
     def print_statistics(self):
+        field_size = 60
+        top_rss = 10
+        results_sorted = []
+
+        # prepare standalone subpath '<vardir>/statistics' for statistics files
+        stats_dir = os.path.join(Options().args.vardir, 'statistics')
+        safe_makedirs(stats_dir)
+
+        # RSS
+
+        # print to stdout RSS statistics for all failed tasks
+        if self.failed_tasks:
+            color_stdout('RSS of the failed tasks:\n', schema='test_var')
+            for task in self.failed_tasks:
+                task_name = self.get_failed_task_name(task)
+                if task_name in self.rss_results:
+                    color_stdout('* %6.1f %s %s\n' % (float(self.rss_results[task_name]) / 1024,
```

Comment:
Can we show units (like `1.23M`)?

Reply:
Do we really need it ? We have already printed it in the header of the column:
"Up to 10 most RSS used tasks in Mb (may skip tests run faster than 0.1 sec):"
also we collect results only in Mb and it will be constant prefix, so I don't think we need it.

Reply:
I'd missed it, okay.